cern_access/indico_cern_access/forms.py

```diff
@@ -105,6 +106,16 @@ class AccessIdentityDataForm(IndicoForm):
     birth_date = IndicoDateField(_('Birth date'), [DataRequired()])
     nationality = SelectField(_('Country of birth'), [DataRequired()])
     birth_place = StringField(_('Place of birth'), [DataRequired()])
+    by_car = BooleanField(_('Are you bringing your own car?'), [Optional()], widget=SwitchWidget(_('Yes'), _('No')),)
```

Comment:
Do we need those `Yes` and `No` here because of  `_` being different from the indico's `_`? (I am asking since those labels are default for the `SwitchWidget`)